docs/source/get_started/access_info/access_info_css.ipynb

```diff
@@ -84,7 +84,7 @@
     {
```

Comment:
<p>With <a href="https://github.com/Infleqtion/client-superstaq/pull/743" target="_blank">https://github.com/Infleqtion/client-superstaq/pull/743</a> merged in, the prefered use of <code>input_circuit()</code> and <code>compiled_circuit()</code> (particularly in <code>css</code> to avoid deprecation warning) is with an index arg, i.e., <code>input_circuit(index=0)</code> or simply <code>input_circuit(0)</code> to retrieve a single circuit (or the first circuit if the user passed a list of circuits) </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Infleqtion/client-superstaq/pull/728/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>